src/chains/algorand_1/algoCreateAccount.ts

```diff
@@ -96,46 +107,41 @@ export class AlgorandCreateAccount implements CreateAccount {
     notSupported('CreateAccount.composeTransaction')
   }
 
+  // TODO: Support recycling & alreadyExists
   /** Determine if desired account name is usable for a new account.
+   * Recycling is not supported for now. Will be supported in the future.
    */
-  async determineNewAccountName(): Promise<any> {
-    // TODO Algo
-    notSupported('CreateAccount.determineNewAccountName')
+  async determineNewAccountName(accountName: AlgorandEntityName): Promise<any> {
+    return { alreadyExists: false, newAccountName: accountName, canRecycle: false }
   }
 
-  /* Not supported for Algorand */
-  async generateAccountName(): Promise<any> {
-    // TODO Algo
-    notSupported('CreateAccount.generateAccountName')
-    return null
+  /** Returns a the Algorand Address as AlgorandEntityName brand for the public key provide in options -
```

Comment:
multiple typos in comment
The brand is an internal thing. You never refer to brand as a type. So this should say returns AlgorandEntityName (not brand)